scripts/pipeline/orthology_benchmark.py

```diff
@@ -628,6 +628,68 @@ def calculate_goc_genes(gene1: str, gene2: str, neighbourhood1: pandas.DataFrame
     return (shared_orthologs / (2 * n)) * 100
 
 
+def calculate_goc_genomes(core_name1: str, core_name2: str, gtf_dir: str, res_dir: str, n: int, m: int)\
```

Comment:
I wonder if it could be worth providing default values for `n` and `m` in this function? This could be a useless suggestion because I cannot yet picture how this will be used - and I imagine it to be dynamic in the benchmarking use-case - in which if always dynamic a default value would be useless?

Reply:
In fact, `n=2` by default in function `calculate_goc_genes()` so there is no reason for not having it here as well. I think I'll go with `n=2, m=3` 👍 Thanks 🙂